### DISCUSSION TITLE
Android res/mipmap-ldpi-v4 icon size

### DISCUSSION THREAD

**Discussion by sesisoft-worldflipper** [airsdk/Adobe-Runtime-Support#983] [Q&A]
Hello.
I'm using version 33.1.1.533.
ADT generated .apk well.
In the generated .apk, the icon size of res/mipmap-ldpi-v4/icon.png is 48x48.
According to Android document, the icon size must be 32x32.
Please tell me How to change the icon size.

**Comment by ajwfrost** [comment]
Actually AIR will be looking for a 36x36 sized icon to put into the ldpi folder, and if it doesn't find one that's this size then it will choose the next-largest.
36x36 is from the table in https://developer.android.com/training/multiscreen/screendensities
> 36x36 (0.75x) for low-density (ldpi)

Are you able to double-check, if you can point to documentation that says it should be 32x32 then we can investigate further..
